templates/solutions/ai/index.html

```diff
@@ -0,0 +1,571 @@
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/206a6c8b-apache-spark-logo.png",
+            alt="Spark",
+            width="149",
+            height="257",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+          <hr class="p-rule"/>
+          <p class="u-align-text--left">
+            <a href="/data/spark" aria-label="Learn more about Apache Spark">Learn more&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/14bcf386-mlflow-logo.png",
+            alt="MLflow",
+            width="183",
+            height="257",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+          <hr class="p-rule"/>
+          <p class="u-align-text--left">
+            <a href="https://ubuntu.com/ai/mlflow" aria-label="Learn more about MLflow">Learn more&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/d634e1dd-opensearch-logo.png",
+            alt="OpenSearch",
+            width="321",
+            height="257",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+          <hr class="p-rule"/>
+          <p class="u-align-text--left">
+            <a href="/data/opensearch" aria-label="Learn more about OpenSearch">Learn more&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/20df0dfc-kafka.png",
+            alt="Kafka",
+            width="228",
+            height="257",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+          <hr class="p-rule"/>
+          <p class="u-align-text--left">
+            <a href="/data/kafka" aria-label="Learn more about Apache Kafka">Learn more&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/71501f0a-mongo-db-logo.png",
+            alt="MongoDB",
+            width="213",
+            height="257",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+          <hr class="p-rule"/>
+          <p class="u-align-text--left">
+            <a href="/data/mongodb" aria-label="Learn more about MongoDB">Learn more&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+        <div class="p-logo-section__item u-no-padding--right">
```

Comment:
Was it supposed to be `u-no-margin--right` (to remove margin from last item)? Removing padding doesn't seem to have any effect.

But it seems that removing margin is also not needed, as all logos seem to fit the width of the page.